.circleci/config.yml

```diff
@@ -34,24 +34,33 @@ commands:
           - v1-deps-{{ checksum "build.sbt" }}
 
       - run:
-          name: Compile
-          command: sbt test:compile
+          name: Compile 2.11
+          command: sbt ++2.11.12 test:compile
+
+      - run:
+          name: Compile 2.12
+          command: sbt ++2.12.11 test:compile
 
       - save_cache:
           paths:
           - ~/.ivy2
           key: v1-deps-{{ checksum "build.sbt" }}
 
       - run:
-          name: Run Tests
-          command: sbt coverage test coverageReport
+          name: Test 2.11
+          command: sbt ++2.11.12 coverage test coverageReport
+
+      - run:
+          name: Test 2.12
+          command: sbt ++2.12.11 coverage test coverageReport
 
       - run:
           when: always
           name: Gather results
           command: |
             mkdir -p results/{tests,coverage}
             cp **/target/test-reports/*.xml results/tests/
+            # TODO: save results for 2.11 as well
```

Comment:
I need to talk with @evbruno to figure out the best way to do so.

Reply:
This will be handled in [DRV-222](https://faunadb.atlassian.net/browse/DRV-222).